browser-versions.json

```diff
@@ -1,4 +1,4 @@
 {
-  "chrome:beta": "102.0.5005.40",
+  "chrome:beta": "102.0.5005.49",
   "chrome:stable": "101.0.4951.64"
```

Comment:
@jennifer-shehane curious, should we be doing something similar with firefox? Seems like there's been quite a few firefox issues lately we maybe would have caught before our users if we had a similar approach.